.ci/hardware.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+set -e -v
+
+NAME=$0
+COMMAND=$1
+
+if [[ "$COMMAND" == "install" ]]; then
+    pip install coverage  # needed for the uploading in after_success
+    openssl aes-256-cbc -K $encrypted_e0365add3e93_key -iv $encrypted_e0365add3e93_iv -in .ci/travis_rsa.enc -out ~/.ssh/id_rsa -d
+    chmod 600 ~/.ssh/id_rsa
+    echo -e "$INTELHOST_INFO" >> ~/.ssh/config
+    ssh -o StrictHostKeyChecking=no loihihost "echo 'Connected to loihihost'"
+    scp -r . loihihost:/tmp/nengo-loihi-$TRAVIS_JOB_NUMBER
+elif [[ "$COMMAND" == "script" ]]; then
+    ssh loihihost << EOF
+        sh /etc/profile
+        sh ~/.bashrc
+        cd /tmp/nengo-loihi-$TRAVIS_JOB_NUMBER
+        conda create -y -n travis-ci-$TRAVIS_JOB_NUMBER python=3.5.2 scipy
+        source activate travis-ci-$TRAVIS_JOB_NUMBER
+        pip install -e .[tests]
+        pip install ~/travis-ci/nxsdk-0.7.5.tar.gz
+        SLURM=1 coverage run -m pytest --target loihi --no-hang -v --duration 50 --color=yes && \
+        coverage report -m && \
+        coverage xml
```

Comment:
I believe keeping continuation lines at the same indent level is the bash convention (although I am not an expert).

Reply:
Does it have to be our convention?

Reply:
It doesn't have to, but I prefer it as well :smile:

Reply:
I guess these are really independent commands, so maybe it does make sense to have no indent. If we were using pipes `|` instead of `&&` I might argue for intents more.

Reply:
I also prefer indenting, but yeah since we're doing `&& \` at the end I'm fine with same level. Though I also prefer the binary operator on the following line ;) Anyway, not worth blocking.